v2/store/core.ts

```diff
@@ -50,6 +51,34 @@ const selectEnabledJars = (state: RootState) => {
     (jar) => jar.enablement === AssetEnablement.ENABLED,
   );
 };
+const selectMaxApy = (state: RootState) => {
```

Comment:
@larrythecucumber321 just heads up that Redux has a nice support for memoized selectors so even complex selectors rerender/run only when data changes ([more info](https://redux.js.org/usage/deriving-data-selectors#optimizing-selectors-with-memoization) on that). No need to worry about it for now, I'll start adding them soon.

Reply:
Look forward to seeing your solution on this!